### DISCUSSION TITLE
UV support

### DISCUSSION THREAD

**Discussion by stonegray** [spensbot/captivate#45] [Ideas]
Currently there appears no way to specify UV in 1.0.0. I've tried setting it as a cyan or magenta channel, but the color picker doesn't seem to let me use only that color.

This also occurs to a lesser extent with amber; there's about 30/255 leakage of other channels making the orange appear washed out. 

Color is complicated and I don't have a good solution, perhaps being able to switch a scene into a mode with sliders instead of a palate might be a temporary solution? I'm not clear how blending from/to UV should even work so perhaps keeping it separate makes sense.

**Comment by NicholasTracy** [comment]
I know this is a 1 year old post, but since not much has happened in development. I want to chime in, Sound Switch is where I am currently coming from and it also struggles with Actual white, Amber, and UV channels. It currently handles these as separate sliders next to the color wheel for the RGB channels and then you add them as custom color pallet mixes. I guess this would be the best way currently to handle it until something else is figured out.
